src/components/migrated-legacy.js

```diff
@@ -155,9 +161,9 @@ class MigratedBchRecoveryForm extends Component {
     const { bitgo } = this.props;
     this.setState({ error: '', recovering: true });
 
-    const bch = bitgo.coin('bch');
-    const bchWallets = await bch.wallets().list();
-    const migratedWallet = _.find(bchWallets.wallets, w => w._wallet.migratedFrom === this.state.walletId);
+    const coin = bitgo.coin(this.props.bitgo.env === 'prod' ? this.state.coin : `t${this.state.coin}`);
+    const wallets = await coin.wallets().list();
+    const migratedWallet = _.find(wallets.wallets, w => w._wallet.migratedFrom === this.state.walletId);
```

Comment:
is there a cleaner way to do this?  ie: why fetch the whole list when all we're looking for is a single `migratedWallet`? 

could this be condensed into something like:

```javascript
migratedWallet = await coin.wallets(migratedFrom=this.state.WalletId);
```
(i'm not familiar w/ the orm to know if the above will actually work - but you get the gist)

Reply:
so, ```state.walletId``` is actually the v1 BTC wallet. We could do something like what you mentioned, but that would require us to add an additional field in the UI (or change the current field in the UI). It actually might make more sense to change the current field in the UI. I'll take a look tomorrow